cmd/funnel/main.go

```diff
@@ -29,14 +33,14 @@ func main() {
 	})
 	flag.Parse()
 
-	nodeConfig, err := config.NewNodeConfig()
-	if err != nil {
-		log.Fatal().Err(err).Msg("error loading node config")
+	tsAuthKey := utils.GetEnvString(tsAuthKeyVar, "")
```

Comment:
actually i think we can just remove this entirely. looks like tsnet already checks for this env var: https://pkg.go.dev/tailscale.com/tsnet#Server